xds/src/main/java/io/grpc/xds/XdsClusterResource.java

```diff
@@ -382,6 +382,11 @@ static boolean hasNegativeValues(Duration duration) {
     return upstreamTlsContext;
   }
 
+  private static boolean isSystemRootCertsEnvVarSet() {
```

Comment:
The result should be stored in a `static` or be a field on the object. This class is essentially a singleton, so those are mostly the same thing... although I see that the class doesn't have a private constructor, so a test could create a different instance. Probably best not to rely on that bug.

Reply:
Done.